controllers/watermarkpodautoscaler_controller.go

```diff
@@ -460,6 +460,7 @@ func setStatus(wpa *datadoghqv1alpha1.WatermarkPodAutoscaler, currentReplicas, d
 	if rescale {
 		now := metav1.NewTime(time.Now())
 		wpa.Status.LastScaleTime = &now
+		wpa.Status.ScalingEventsCount += 1
```

Comment:
Do we ever reset this to 0 or does it keep increasing? Should it be reset when WPA is modified?

Reply:
I agree that we should revisit this. Though it should be applied to the entire status. I will add a card to review whether we should improve on it.